code-challenges.test.js

```diff
@@ -1,52 +1,165 @@
-// ASSESSMENT 2: Coding Practical Questions with Jest
+// // ASSESSMENT 2: Coding Practical Questions with Jest
 
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
+// const { Console } = require("console")
 
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
+// // Please read all questions thoroughly
+// // Pseudo coding is REQUIRED
+// // If you get stuck, please leave comments to help us understand your thought process
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+// // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// // a) Create a test with expect statements for each of the variables provided.
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+describe("object1", () => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+      expect(object1()).toEqual("15 is divisible by three")
+    })
+  })
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+describe("object1", () => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+      expect(object2()).toEqual("0 is divisible by three")
+    })
+  })
 
-// a) Create a test with expect statements for each of the variables provided.
+describe("object3", () => {
```

Comment:
Since you are creating a test for only one function, you only need to have one describe and one it block to test the function.  You can then provide multiple expect statements for each test variable provided.